docs/docs/how-tos/index.md

```diff
@@ -9,16 +9,24 @@ Here you’ll find answers to “How do I...?” types of questions. These guide
 
 ## LangGraph
 
-### Controllability
+### Graph API Basics
 
-LangGraph offers a high level of control over the execution of your graph.
+- [How to define and update graph state](state-reducers.ipynb)
+- [How to create a sequence of steps](sequence.ipynb)
+- [How to create branches for parallel execution](branching.ipynb)
+- [How to create and control loops with recursion limits](recursion-limit.ipynb)
```

Comment:
we have different names for this one -- i kind of like the shorter version `How to create and control loops`